### PR TITLE
[tempo-distributed] Fixed comments

### PR DESCRIPTION
**Pull request by xogoodnow**
Fixed a few issue within the comments

### CODE REVIEW

File: charts/tempo-distributed/README.md [hunk 1]
```diff
@@ -361,8 +361,8 @@ The memcached default args are removed and should be provided manually. The sett
 | distributor.tolerations | list | `[]` | Tolerations for distributor pods |
 | distributor.topologySpreadConstraints | string | Defaults to allow skew no more then 1 node per AZ | topologySpread for distributor pods. Passed through `tpl` and, thus, to be configured as string |
 | enterprise.enabled | bool | `false` |  |
-| enterprise.image.repository | string | `"grafana/enterprise-traces"` | Grafana Enterprise Metrics container image repository. Note: for Grafana Tempo use the value 'image.repository' |
-| enterprise.image.tag | string | `"v2.6.1"` | Grafana Enterprise Metrics container image tag. Note: for Grafana Tempo use the value 'image.tag' |
+| enterprise.image.repository | string | `"grafana/enterprise-traces"` | Grafana Enterprise Tetrics container image repository. Note: for Grafana Tempo use the value 'image.repository' |
```
Comment (zalegrala) [inline charts/tempo-distributed/README.md:364]: I think this was meant to be `Traces` not `Tetrics`.

File: charts/tempo-distributed/README.md [hunk 2]
```diff
@@ -795,9 +795,9 @@ The memcached default args are removed and should be provided manually. The sett
 | queryFrontend.query.extraVolumeMounts | list | `[]` | Extra volumes for tempo-query pods |
 | queryFrontend.query.extraVolumes | list | `[]` | Extra volumes for tempo-query deployment |
 | queryFrontend.query.image.pullSecrets | list | `[]` | Optional list of imagePullSecrets. Overrides `tempo.image.pullSecrets` |
-| queryFrontend.query.image.registry | string | `nil` | The Docker registry for the query-frontend image. Overrides `tempo.image.registry` |
-| queryFrontend.query.image.repository | string | `"grafana/tempo-query"` | Docker image repository for the query-frontend image. Overrides `tempo.image.repository` |
-| queryFrontend.query.image.tag | string | `nil` | Docker image tag for the query-frontend image. Overrides `tempo.image.tag` |
+| queryFrontend.query.image.registry | string | `nil` | The Docker registry for the tempo-query image. Overrides `tempo.image.registry` |
+| queryFrontend.query.image.repository | string | `"grafana/tempo-query"` | Docker image repository for the tempo-queryimage. Overrides `tempo.image.repository` |
```
Comment (zalegrala) [inline charts/tempo-distributed/README.md:799]: `tempo-query image`

**Review by zalegrala — COMMENTED** [review]
Thanks for the PR.  A couple of comments.

File: charts/tempo-distributed/README.md (hunk 1 above)
Comment (xogoodnow) [reply charts/tempo-distributed/README.md:364]: Oh my bad.
Good catch.

**Review by mapno — APPROVED** [review]

**Pull request merged**